File: Foundation/Patterns/countingtriangle.cpp
#include<iostream>
using namespace std;

void countTriangle(int row)
{
    int nst = 1;
    int count = 1;
    for(int r = 1; r <= row; r++)
    {
        for(int cst = 1; cst <= nst; cst++)
        {
            cout<<count<<"\t";
            count++;
        }
        
        nst++;
        cout<<endl;
    }
}

int main()
{
    int n;
    cin>>n;
    countTriangle(n);
    return 0;
}

File: Foundation/Patterns/countingtriangle.java
import java.util.Scanner;
public class countingtriangle
{
    public static Scanner scn = new Scanner(System.in);
    public static void countingtriangle(int row)
    {
        int nst = 1;
        int count = 1;
        for(int r = 1; r <= row; r++)
        {
            for(int cst = 1; cst <= nst; cst++)
            {
                System.out.print(count +  "\t");
                count++;
            }
            nst++;
            System.out.println();
        }
    }

    public static void main(String[] args)
    {
        int n = scn.nextInt();
        countingtriangle(n);
    }
}